scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -516,24 +512,6 @@ <h5 class="modal-title" id="exampleModalLabel">Edit synopsis</h5>
   </form>
 {% endmacro %}
 
-{% macro remove_form(url, hidden_input=None, button_name=None, button_value=None) %}
```

Comment:
This is not in use as it is a duplicate of the remove_form imported from cases/utils I believe?

Reply:
Looks like it!